src/components/ActiveVideo/ActiveVideo.tsx

```diff
@@ -115,11 +115,8 @@ const ActiveVideo: FunctionComponent<Props> = (props) => {
   )
 }
 
-export default trackComponent(
-  localised(
-    withCrossDeviceWhenNoCamera(
-      withFailureHandling(withPermissionsFlow(ActiveVideo))
-    )
-  ),
-  'face_liveness'
+export default localised(
```

Comment:
For some reason, GitHub prevents me from adding comments on lines that are not part of the diff, but there's a warning line 29 you can address.

Reply:
You're absolutely right. Unused import for track_component, my bad. Fixed now!